flocker/node/_loop.py

```diff
@@ -264,9 +264,14 @@ def output_CONVERGE(self, context):
         d = self.deployer.discover_local_state()
 
         def got_local_state(local_state):
+            # Update cluster_state here and then no need to pass local state to calculate_necessary_state_changes.
```

Comment:
Remove these comments, add back comment that used to be in P2PNodeDeployer about why we're doing this.